backend/remark_at_building/views.py

```diff
@@ -121,11 +125,39 @@ class AllRemarkAtBuilding(APIView):
     permission_classes = [IsAuthenticated, IsAdmin | IsSuperStudent]
```

Comment:
Could the permission, isStudent be added, a student should be able to retrieve the remarks of the buildings he was at (`StudentOnTour`)?

Reply:
Fixed